app/views/create-from-url.html

```diff
@@ -15,24 +15,9 @@ <h2>Create a New Project</h2>
               </div>
               <div ng-if="!noProjects && !canCreateProject">
                 <h2>Choose a Project</h2>
-                <ui-select ng-model="selected.project">
-                  <ui-select-match placeholder="Project name">
-                    {{$select.selected | displayName}}
-                  </ui-select-match>
-                  <ui-select-choices repeat="project in projects | searchProjects : $select.search">
-                    <span ng-bind-html="project | displayName | highlightKeywords : $select.search"></span>
-                    <span ng-if="project | displayName : true" class="small text-muted">
-                      &ndash;
-                      <span ng-bind-html="project.metadata.name | highlightKeywords : $select.search"></span>
-                    </span>
-                  </ui-select-choices>
-                </ui-select>
-              </div>
-              <div ng-if="!noProjects && canCreateProject">
-                <uib-tabset>
-                  <uib-tab>
-                    <uib-tab-heading>Choose Existing Project</uib-tab-heading>
-                    <ui-select ng-model="selected.project">
+                <ng-form name="forms.selectProjectForm">
```

Comment:
Use `<form>` here instead of `<ng-form>` if it's not already nested in another form element.